xmtp_db/src/encrypted_store/key_package_history.rs

```diff
@@ -52,7 +52,7 @@ impl DbConnection {
         &self,
         id: i32,
     ) -> Result<Vec<StoredKeyPackageHistoryEntry>, StorageError> {
-        let result = self.raw_query_read(|conn| {
+        let result = self.raw_query_read::<_, StorageError, _>(|conn| {
```

Comment:
I couldn't find a good ergonomic way to keep eliding the error type with the new `ConnectionExt` trait. Maybe a future refactor can consider how to do this

Reply:
I think we can get around this with a PhantomData where we can define the default type. Something like

```rust
#[doc(hidden)]
pub struct DbConnection<C = crate::DefaultConnection, E = StorageError> {
    conn: C,
    _phantom: PhantomData<E>,
}
```

Reply:
Created a PR https://github.com/xmtp/libxmtp/pull/1986